Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -482,8 +529,9 @@ def send_slack_request_sync(client: slack_sdk.WebClient, method: str, http_verb:
         try:
             demisto.debug(f'Sending slack {method} (sync). Body is: {str(body)}')
             if http_verb == 'POST':
-                if file_:
-                    response = client.api_call(method, files={"file": file_}, data=body)
+                if file_upload_params:
+                    # Use three-stage file_upload_v2 'wrapper' method in SDK client class in case there are file_upload_params
```

Comment:
```suggestion
                    # When file_upload_params is provided, performs a three-stage `file_upload_v2` wrapper method in Slack's SDK client
```
Worth adding a link to Slack's docs about this change